.storybook/main.ts

```diff
@@ -0,0 +1,25 @@
+import type { StorybookConfig } from '@storybook/react-webpack5';
+
+const config: StorybookConfig = {
+	stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
+	addons: [
+        '@storybook/addon-webpack5-compiler-babel',
+        '@storybook/addon-essentials',
+        '@storybook/addon-themes',
+        '@storybook/addon-interactions',
+        '@storybook/test',
+        '@storybook/blocks',
+        '@storybook/react',
+        '@storybook/react-webpack5',
+        'storybook-dark-mode',
+        '@storybook/theming'
```

Comment:
We might want to use dark-reader as theme toggler, since it is the one used also in our carbonio environment. We can open a separate issue to better integrate it

Reply:
I'll create a ticket then.